keanu-project/src/main/java/io/improbable/keanu/vertices/intgr/nonprobabilistic/operators/binary/IntegerMaxVertex.java

```diff
@@ -1,17 +1,19 @@
 package io.improbable.keanu.vertices.intgr.nonprobabilistic.operators.binary;
 
 import io.improbable.keanu.tensor.intgr.IntegerTensor;
+import io.improbable.keanu.vertices.LoadParentVertex;
+import io.improbable.keanu.vertices.SaveableVertex;
 import io.improbable.keanu.vertices.intgr.IntegerVertex;
 
-public class IntegerMaxVertex extends IntegerBinaryOpVertex {
+public class IntegerMaxVertex extends IntegerBinaryOpVertex implements SaveableVertex {
 
     /**
      * Finds the maximum between two vertices
      *
      * @param left  one of the vertices to find the maximum of
      * @param right one of the vertices to find the maximum of
      */
-    public IntegerMaxVertex(IntegerVertex left, IntegerVertex right) {
+    public IntegerMaxVertex(@LoadParentVertex(A_NAME) IntegerVertex left, @LoadParentVertex(B_NAME) IntegerVertex right) {
```

Comment:
`left` and `right` here! Let's keep this consistent with the rest of the ints

Reply:
Done